### PR TITLE
refactor-signalAPI-javadoc

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java [hunk 1]
```diff
@@ -147,6 +147,7 @@
 
     /**
      * Remove the given {@code signal} from the set of job signals
+     * In order to remove a signal that was not yet received, the signalsSubSet parameter should contain "_ready" prefix and the signal name
```
Comment (mboussaa) [inline scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java:150]: the "_ready" prefix

**Review by mboussaa — APPROVED** [review]

**Review by tao-xinxiu — APPROVED** [review]

File: scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java (hunk 1 above)
Comment (fviale) [reply scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java]: ready suffix, no ?

Comment (oanaBiancaSchiopu) [reply scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java]: it's prefix, but it shoud be "ready_" not "_ready", I will change it. Thank you

Comment (fviale) [reply scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java]: ah ok

Comment (oanaBiancaSchiopu) [reply scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java]: it's before the signal name: signalapi.removeSignal("ready_Stop_Job")

**Pull request merged**